_posts/2022-10-14-mdas-new.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "New Repo: MDAS"
+categories: new-repo
+---
+
+The [MSU Disentanglement Analysis Software (MDAS)](https://github.com/LLNL/MDAS) is used to disentangle the forced and unforced components of tropospheric temperature change over the satellite era (after 1979) using maps of surface temperature change as a predictor. Input data is available via Zenodo ([doi: 10.5281/zenodo.7199961](https://zenodo.org/record/7199961#.Y2AQNHbMKbg)). An accompanying publication is forthcoming from the *Proceedings of the National Academy of Sciences*.
```

Comment:
Should we link to https://www.pnas.org/ for the publication? (Maybe not until it's published?)

Reply:
It's not published yet